janusgraph-solr/src/test/resources/solr/core-template/schema.xml

```diff
@@ -547,6 +547,7 @@
                prefixTree="geohash"/>
```

Comment:
Can you do me a favor and replace `org.locationtech.spatial4j.context.jts.JtsSpatialContextFactory` with `com.spatial4j.core.context.jts.JtsSpatialContextFactory` above? Otherwise I can open another PR with the fix. In testing this with Solr 5.5.4 (see [TESTING.md](https://github.com/JanusGraph/janusgraph/blob/master/TESTING.md#running-solr-tests-with-external-server)) I realized my change to the test schema in #400 broke backwards-compatibility. Incidentally the tests on this branch against Solr 5.5.4 all passed.